uwblueprint/rowan-house
# Fix(es): outstanding feedback on course preview block

## Implementation Description
- Some outstanding changes building on https://github.com/uwblueprint/rowan-house/pull/95
  - [x] Add courseId, moduleId to specify route link for `ModulePreview` component
  - [x] Remove `ModulePreview` text highlight on hover
  - [x] Migrate dummy data out of specific components
  - [x] Extend horizontal line under courses / create new course title
  - [x] Moved `EditActionsKebabMenu` from "common" to "admin" since I didn't see it used elsewhere?

TBD:
- need to figure out how to make Tag sit at base of the ModulePreview box no matter length of title

## Screenshots
https://user-images.githubusercontent.com/40974372/158976783-c753f0fa-f063-4184-85b6-9a437af41ebd.mov

## What should reviewers focus on?
- make sure all changes make sense, suggestions for TBD item very welcome

## Testing Procedure
- What test(s) should we run to make sure your code works?
   - What ways could the input be wrong?
   - Is it secure? Could nefarious users access/break it?

Visual QA at the moment

## Things to Note
- Additional dependencies/libraries you've added?
- Things you'd want to know when coming back to the code after a few months

n/a
easily testable?

## Checklist
